ui.R

```diff
@@ -2,99 +2,124 @@ library(shiny)
+  skin = "purple",
+  dashboardHeader(title = "Hypothesis Testing", titleWidth = 250,
+                  tags$li(class = "dropdown", actionLink("info", icon("info"))),
+                  tags$li(class = "dropdown",
+                          tags$a(href='https://github.com/EducationShinyAppTeam/BOAST',
+                                 icon("github"))),
+                  tags$li(class = "dropdown",
+                          tags$a(href='https://shinyapps.science.psu.edu/',
+                                 icon("home")))
+  ),
+  
   # Sidebar
   dashboardSidebar(
-    width = 260,
+    width = 250,
     sidebarMenu(
       id = "tabs",
-      menuItem("Overview", tabName = "overview", icon = icon("dashboard")),
-      menuItem("Filtering", tabName = "filtering", icon = icon("wpexplorer")),
-      menuItem("Hypothesis Testing", tabName = "hypothesis_testing", icon = icon("cogs")),
-      menuItem("Data", tabName = "data", icon = icon("table"))
+      menuItem("Overview", tabName = "Overview", icon = icon("dashboard")),
+      menuItem("Explore", tabName = "Explore", icon = icon("wpexplorer")),
+      menuItem("Challenge", tabName = "Challenge", icon = icon("cogs")),
+      menuItem("Data", tabName = "Data", icon = icon("table")),
+      menuItem("References", tabName = "References", icon = icon("leanpub"))
+    ),
+    tags$div(
+      class = "sidebar-logo",
+      boastUtils::psu_eberly_logo("reversed")
     )
   ),
-
+  
   # Content within the tabs
   dashboardBody(
     tags$head(
-      tags$link(rel = "stylesheet", type = "text/css", href = "Feature.css")
+      tags$link(rel = "stylesheet", type = "text/css",
+                href = "https://educationshinyappteam.github.io/Style_Guide/theme/boast.css")
     ),
+    # This is contents for the first tab, "Overview"
     tabItems(
       tabItem(
-        tabName = "overview",
-        tags$a(href = "http://stat.psu.edu/", tags$img(src = "PS-HOR-RGB-2C.png", align = "left", width = 180)),
-        br(), br(), br(),
-
-        h3(strong("About:")),
-        h4("In this app the goal is to learn about the reasoning of a hypothesis test about proportions."),
-        h4("This app uses 2016-2017 data for the NBA regular season."),
-        br(),
-        h3(strong("Instructions:")),
-        h4(tags$li("In Part 1 you will look at how the population distribution of all the players' free throw percentages is affected by filtering (restricting attention to a subpopulation).")),
-        tags$head(
-          tags$style(HTML("#go{background-color: #367fa9}"))
-        ),
+        tabName = "Overview",
+        withMathJax(),
+        h1(strong("Hypothesis Test for Means")),
+        p("In this app the goal is to learn about the reasoning of a hypothesis test about proportions."),
+        h2("Instructions"),
+        tags$li("In Part 1 you will look at how the population distribution of all the players' free throw percentages is affected by filtering (restricting attention to a subpopulation)."),
```

Comment:
Break long lines in your code to improve readability. You can put a return in after "free" and before "throw" to move the line down. The app will still display the text coherently but now your code is easier to read.